libs/contracts/README.md

```diff
@@ -1,3 +1,19 @@
-# FeatureBoard API Contracts
+# FeatureBoard Contracts
```

Comment:
Could be cool to add some badges to our packages

eg https://shields.io/badges/npm-scoped for the latest version NPM package.